rfcs/0013-deprecate-currency-factory.md

```diff
@@ -0,0 +1,210 @@
+depreciating Currency Factory and its replacements.
+
+## Reasoning
+
+There are several reasons we stand to benefit from replacing Currency Factory.
+These reasons are detailed below:
+
+### Currency Factory Ranges are Confusing
+  
+The Currency Factory ranges are a confusing abstraction that we don't 
+stand to gain from. We really only have two types of currencies native 
+(mintable) and non-native (external). However, we currently have many more 
+ranges that enforce artificial limitations on various abstractions of these two 
+currency types. This complicates the currency creation process and leads to 
+confusion while providing no benefit to us.
+
+### No Permissionless Asset Creation
+
+> We do not have permissionless asset (non-payable/no sufficient) - it is 
+impossible to create no marketable assets (which is needed for 
+governance/airdrops).
+
+### Currency Factory is Already Losing Functionality
+
+Through RFCs 0009 and 0010, we decided to move away from storing ED in Currency 
+Factory. After the removal of ED storage, Currency Factory was left with two 
+uses: reserving asset IDs and storing asset metadata. 
+
+As discussed, Currency Factory overcomplicates the asset ID reservation system. 
+As for asset metadata, we don't have standards for formatting this nor do we 
+utilize it. Assuming we replace the reservation system, Currency Factory will be 
+left without value.
+
+## Requirements
+
+If we are to deprecate Currency Factory, we will need to ensure the requirements
+it fulfills are still met by some solution.
+
+* Permissioned asset creation can be done without collision in the asset ID
+
+* Asset metadata is available
+
+Additionally, Currency Factory failed to meet some requirements that we should
+enforce in a future solution.
+
+* Permissionless asset creation can be done without collision in the asset ID
+
+## Solution Proposals
+
+There are multiple ways we could go about replacing Currency Factory. Possible 
+solutions are not necessarily exclusive (i.e. we could implement multiple). 
+These solutions are detailed below:
+
+### Simplify & Move the Asset ID Reservation System
+
+One way to remove the need for pallet-currency-factory is to move its 
+functionality to pallet-asset-registry. If we also simplify the asset ID 
+reservation system, this will be a minimal change.
+
+This keeps the functionality of Currency Factory around but removes the need for
+another pallet. This does not enable premissionless asset creation without the
+need for more design.
+
+#### Technical Implementation
+
+Instead of reserving asset IDs via our current range system, we could simply 
+have a nonce for reserving new asset IDs while ensuring a lack of collisions. 
+This nonce can start at an arbitrary but high number so that our hard-coded 
+asset IDs are still safe. This would reduce the complexity of our current 
+reservation system while still avoiding collision.
```

Comment:
```suggestion
Instead of reserving asset IDs via our current range system, we could simply 
have one nonce for reserving new asset IDs, which would (by the definition of a nonce) ensure no collisions. 
This nonce can start at an arbitrary but high number so that our hard-coded 
asset IDs are still safe. This would reduce the complexity of our current 
reservation system while still avoiding collision.
```